js/cards.js

```diff
@@ -198,6 +201,56 @@ export function updateGachaView(e) {
       .classList.remove("visually-hidden");
   }
 }
+export function updateGachaMotion(){
+  // get the list of cards
+  let cardImageList = document.getElementsByClassName("card-image");
+  let cardList = document.getElementsByTagName("tcg-card");
+  // document.getElementsByClassName("card-component");
```

Comment:
delete commented out code

Reply:
Oops, there's always something! Removed this.